src/api/lib/tasks/dev.rake

```diff
@@ -249,6 +249,65 @@ namespace :dev do
     end
   end
 
+  # Run this task with: rails dev:workflow_runs:data"
+  namespace :workflow_runs do
+    desc 'Creates workflow runs, artifacts and all its dependencies'
+    task data: :environment do
+      unless Rails.env.development?
+        puts "You are running this rake task in #{Rails.env} environment."
+        puts 'Please only run this task with RAILS_ENV=development'
+        puts 'otherwise it will destroy your database data.'
+        return
+      end
+
+      require 'factory_bot'
+      include FactoryBot::Syntax::Methods
+
+      puts 'Creating workflow token and workflow runs...'
+
+      admin = User.get_default_admin
+      User.session = admin
+
+      source_project = find_or_create_project(admin.home_project_name, admin)
+      target_project = find_or_create_project('home:Admin:CI:iggy:hello_world:PR-1', admin)
+
+      source_package = source_project.packages.find_by(name: 'hello_world') || create(:package_with_file, name: 'hello_world', project: source_project, file_name: 'README.md', file_content: '# Lorem ipsum')
+      target_package = target_project.packages.find_by(name: 'hello_world') || create(:package_with_file, name: 'hello_world', project: target_project, file_name: 'README.md', file_content: '# Lorem ipsum')
+
+      token = Token::Workflow.find_by(name: 'Testing token') || create(:workflow_token, user: admin, name: 'Testing token')
+
+      ['running', 'success', 'fail'].each do |status|
+        workflow_run = create(:workflow_run, token: token, status: status)
```

Comment:
Would it be possible to also have data in `response_body` and `response_url`?

Reply:
As `response_body` and `response_url` change depending on different statuses, I decided to create factories for workflow runs with different status and refactor the task code a bit.